### ISSUE TITLE
Device(): The config profile (dephy) could not be found.

### ISSUE THREAD

**Issue by imsenthur** [DephyInc/Actuator-Package#108]
Hi Jared,

This error pops up when I try to initialize the device class (and also when I try to run the included demo scripts). Looks like it is unable to download the libs. Your help would be greatly appreciated!

<img width="1209" alt="Screenshot 2023-03-14 at 8 12 41 PM" src="https://user-images.githubusercontent.com/25511437/225172889-f0c456fb-925d-45fa-a592-e2bec148e5d3.png">

I'm running flexsea on Raspbian OS (32bit) with python3.11

**Comment by jcoughlin11** [comment]
@imsenthur Sorry for the delayed response! The necessary C libraries to use `flexsea` are hosted on S3, and by default we try to use access keys stored under a profile called "dephy". However, that bucket should be public, so not needing a profile might work. To fix the issue, though, you'll need a file called: `~/.aws/credentials`  (no extension). In that file is where you can put your access keys. The syntax is:

```shell
[PROFILE]
aws_access_key_id=<YOUR KEY>
aws_secret_access_key=<YOUR KEY>
```

You can have as many profiles as you want. A good idea is to have one called `default` and one called `dephy`. I'll update the README to be clearer about this.

**Comment by Sotilrac** [comment]
Keys are no longer needed, this issue has been resolved in version 10.5.3
